src/components/common/CategorySelect/CategorySelectNode.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { NodeModel } from '@minoru/react-dnd-treeview';
+import { Body } from '../Typography';
+import { ColorPalette } from '@/styles/ColorPalette';
+import ArrowRightIcon from '@/assets/svg/ArrowRightIcon';
+import CategoryIcon from '@/assets/svg/Category';
+import AddIcon from '@/assets/svg/AddIcon';
+import { Input } from '@/components/ui/input';
+import Done from '@/assets/svg/Done';
+
+export type CustomData = {
+  fileType: string;
+  fileSize: string;
+};
+
+type Props = {
+  node: NodeModel;
+  depth: number;
+  isOpen: boolean;
+  onToggle: (id: NodeModel['id']) => void;
+  onAdd: (parentId: NodeModel['id']) => void;
+  onClick: (id: NodeModel['id']) => void;
+  onSubmitNewCategory: (name: string) => void;
+};
+
+const cssStyles = {
+  container: {
+    alignItems: 'center',
+    display: 'grid',
+    gridTemplateColumns: 'auto auto 1fr auto',
+    height: '32px',
+    paddingInlineEnd: '8px',
+    '&:hover': {
+      backgroundColor: ColorPalette.blue['000'],
+    },
+  },
+};
+
+const styles = {
```

Comment:
[질문]

`cssStyles`와 `styles`를 나눈 이유가 있을까요? 불필요하게 나뉜 것 같아서요 ㅎㅎ

Reply:
container는 css랑 style 둘 다 사용하다보니 그냥 모든 스타일을 css에 몰아줬는데, style에 들어갈 스타일과 css에 들어갈 스타일을 나누는 게 좋을까요???

```suggestion
const cssStyles = {
  container: {
    '&:hover': {
      backgroundColor: ColorPalette.blue['000'],
    },
  },
};

const styles = {
  container: {
    alignItems: 'center',
    display: 'grid',
    gridTemplateColumns: 'auto auto 1fr auto',
    height: '32px',
    paddingInlineEnd: '8px',
  }
}
```

Reply:
아니면 그냥 모두 다 styles 객체에 넣고 jsx에서는 style prop이나 css prop에 필요에 따라 넣는 방식이 괜찮을까요?